apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/AddBlock.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import type { TreeBlock } from '@core/journeys/ui/block'
+import { EditorProvider } from '@core/journeys/ui/EditorProvider'
+
+import type { BlockFields_StepBlock as StepBlock } from '../../../../../../../__generated__/BlockFields'
+import {
+  TypographyAlign,
+  TypographyColor,
+  TypographyVariant
+} from '../../../../../../../__generated__/globalTypes'
+import { simpleComponentConfig } from '../../../../../../libs/storybook'
+
+import { AddBlock } from './AddBlock'
+
+const AddBlockStory: Meta<typeof AddBlock> = {
+  ...simpleComponentConfig,
```

Comment:
We use `journeyAdminConfig` here since it has a mobile variant
```suggestion
  ...journeyAdminConfig,
```